### DISCUSSION TITLE
How to redirect an stderr of scoop install cousine-nf-mono if it already exist ?

### DISCUSSION THREAD

**Discussion by hungpham3112** [ScoopInstaller/Scoop#4755] [Help and Questions] [answered]
#### Current Behavior
I add nerd-fonts bucket and are using it for multiple terminal. After uninstalling, I installed both it and python 

`scoop install cousine-nf-mono | scoop install python` 

but it raise the error and skip python. 

![image](https://user-images.githubusercontent.com/75968004/155458265-9a72b683-8436-4ba9-ae93-068e44d8af15.png)

#### Expected Behavior
Hide an error when install `cousine-nf-mono` if it is running in background and continue installation . I don't want to restart machine and run installation again. You can help me with script in powershell.

**Comment by rashil2000** [comment, marked as answer]
From next time please ask the  question in the "Discussions" tab. The issue tracker is for bugs and feature requests only.

**Reply by hungpham3112** [reply]
OK, I'll keep an eye on it
